src/common-utils/devcontainer-feature.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "common-utils",
-    "version": "2.0.5",
+    "version": "2.0.6",
```

Comment:
Isn't this unnecessary since version 2.0.5 doesn't seem to have been released yet?

Reply:
Released 2.0.5! https://github.com/devcontainers/features/pkgs/container/features%2Fcommon-utils

Reply:
Ah, I understand that the maximum number of parallel executions has been reached. Thank you very much.